src/components/fields/location-field/location-helper.ts

```diff
@@ -103,41 +103,60 @@ export namespace LocationHelper {
 		}
 	};
 
+	// Added useFallback arg to handle pin location address.
 	export const fetchAddress = async (
 		query: string,
 		pageNumber: number,
+		useFallback: boolean,
```

Comment:
+ 1

Reply:
Since `formatAddressFromGeocodeInfo` takes in an arg `fallbackToLatLng` that would produce the pin location as the `address`, I was thinking along the lines of declaring `useFallback` to inform the `fetchAddress` function of the `query` that it is handling.

Now, in retrospect, might have been better to handle it in a different function that returns the result list.

Reply:
the parameter for `formatAddressFromGeocodeInfo` is also quite redundant. there are no calls that do not pass in true as the value for `fallbackToLatLng`

if you traced through the code, you can see that the previous change was introduced here
```
if (parsedLocationList.length === 0) {
	const address = `Pin location: ${lat}, ${lng}`;
	return {
		results: [
			{
				address,
				lat,
				lng,
				displayAddressText: address,
			},
		],
	};
}
```
to be able to let user select a pin location

it does not rely on `formatAddressFromGeocodeInfo` as there are no results in `locationList`